service/src/main/resources/db/changelog/changesets/20220412_add_expires_to_ssh_key_pair_table.yaml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
+  - changeSet:
+      id: "add_timestamp_to_ssh_key_pair_table"
+      author: yuhuyoyo
+      changes:
+        - addColumn:
+            tableName: ssh_key_pair
+            columns:
+              - column:
+                  name: expires
+                  type: timestamp
+                  constraints:
+                    nullable: true
+                  remarks: |
+                    Timestamp when the ssh key pair encryption key is expired and the private key is
+                    due for another encryption. This column is null if the key is not encrypted.
```

Comment:
I would call this `lastEncryptedTimestamp` and don't add in the refresh duration as that is a config setting that could change. `expires` is not a good name because it sounds like the key is the thing that is expiring